CHANGELOG

```diff
@@ -1,3 +1,6 @@
+# 4.5.1
```

Comment:
hey @ZIMkaRU , I think this is additional feature so we should dump v4.6.0? cc @ezewer

Reply:
sure)